netspeed/src/netspeed.c

```diff
@@ -1415,7 +1415,7 @@ applet_button_press(GtkWidget *widget, GdkEventButton *event, MateNetspeedApplet
 					GTK_DIALOG_MODAL | GTK_DIALOG_DESTROY_WITH_PARENT,
 					GTK_MESSAGE_QUESTION, GTK_BUTTONS_YES_NO,
 					question,
-					applet->devinfo.name);
+					applet->devinfo->name);
```

Comment:
There is a build warning https://travis-ci.org/github/mate-desktop/mate-applets/jobs/709742708#L4795
```
netspeed.c:1418:6: warning: format not a string literal, argument types not checked [-Wformat-nonliteral]
 1418 |      applet->devinfo->name);
```

Reply:
done

Reply:
Warning is gone , thanks.